sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/ServiceBusEndToEndTests.cs

```diff
@@ -529,10 +425,10 @@ private async Task ServiceBusEndToEndInternal<T>(IHost host = null)
                 $"      \"MaxConcurrentCalls\": {16 * Utility.GetProcessorCount()}",
                 "  }",
                 "  \"SessionHandlerOptions\": {",
+                "      \"AutoComplete\": true,",
                 "      \"MaxAutoRenewDuration\": \"00:05:00\",",
-                "      \"MessageWaitTimeout\": \"00:01:00\",",
-                "      \"MaxConcurrentSessions\": 2000",
-                "      \"AutoComplete\": true",
```

Comment:
Why did this change?

Reply:
I updated to match the correct ordering of the format method.